packages/story-editor/src/components/library/common/searchInput.js

```diff
@@ -58,50 +49,32 @@ function WrappedSearchInput({
   placeholder,
   onSearch,
   disabled = false,
-  incremental = false,
-  delayMs = 500,
 }) {
   // Local state so that we can debounce triggering searches.
   const [localValue, setLocalValue] = useState(initialValue);
```

Comment:
Is saving the `localValue` still necessary? Couldn't we just take the current value of the input for `onSubmit` instead of keeping it in the state with `onChange`?

Reply:
I think we would end up in a state where the search input field doesn't update as the value is passed via the inputValue prop.  Could make further changes to make that work but might be better to handle outside of this PR.